app/src/main/kotlin/com/infinity/foodstandards/ui/authorities/AuthoritiesFragment.kt

```diff
@@ -4,56 +4,44 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
 import androidx.fragment.app.Fragment
-import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModelProvider
 import androidx.navigation.fragment.findNavController
 import com.infinity.foodstandards.R
-import com.infinity.foodstandards.databinding.AuthoritiesFragmentBinding
-import com.infinity.foodstandards.model.LocalAuthoritiesResponse
 import com.infinity.foodstandards.model.LocalAuthority
 
-class AuthoritiesFragment : Fragment(), Observer<LocalAuthoritiesResponse> {
-
-    private var _binding: AuthoritiesFragmentBinding? = null
-    private val binding get() = requireNotNull(_binding) {
-        "Authorities fragment UI accessed after onDestroyView"
-    }
+class AuthoritiesFragment : Fragment() {
```

Comment:
Is it worth removing Fragments altogether and moving this towards a full compose app with single activity? Might make things a bit simpler, but could always be done as a future improvement :)

Reply:
I think my preference would be to update it, it's a bit odd for all of the UI to be compose but still wrapped in Fragments, this is intended for when there's still a mix of XML and Compose in your codebase.

However since it'd require some decent changes and updates to navigation etc, I'm happy for it to be a future improvement to the codebase instead of something done as this PR 👍

Reply:
Thanks